src/main/interface/cli.c

```diff
@@ -3308,6 +3308,9 @@ const cliResourceValue_t resourceTable[] = {
 #if defined(USE_USB_MSC)
     { OWNER_USB_MSC_PIN,   PG_USB_CONFIG, offsetof(usbDev_t, mscButtonPin), 0 },
 #endif
+#ifdef USE_TXPID
+    { OWNER_TXPID,         PG_TXPID_CONFIG, offsetof(txPID_t, auxChannel), 0 },
```

Comment:
What is this doing? `resourceTable` is for pins, not channels.

Reply:
Ah, my mistake. Removed.